### DISCUSSION TITLE
Shadcn/ui for style

### DISCUSSION THREAD

**Discussion by russssl** [basharovV/musicat#12] [Ideas]
There is compoment library called shadcn/ui. it is very nice looking. there is svelte port as well) it could be used as a components style since app is still in beta (https://shadcn-svelte.com/)

**Comment by russssl** [comment]
there is even player example (https://shadcn-svelte.com/examples/music)

**Comment by basharovV** [comment]
Hey thanks for the suggestion! Never heard of this, it's neat that you can copy/paste and customize the components without having to depend on an entire framework. 

I see this working for buttons, menus, input fields, but probably not for the whole app, since there are so many custom bits, and even canvas-based UI (the whole library/queue views for example). It would also need to support themes, which I'm currently working on. 

Before this can happen, though, there is still some work to tidy up existing UI and component-ize things like the volume slider, toggle buttons, etc.
